arduino/OpenMRNLite.h

```diff
@@ -116,6 +117,8 @@ constexpr UBaseType_t OPENMRN_TASK_PRIORITY = ESP_TASK_TCPIP_PRIO - 1;
 
 #endif
 
+OVERRIDE_CONST(gridconnect_bridge_max_incoming_packets, 5);
```

Comment:
should this be moved to OpenMRNLite.cpp or does it not matter that it is included via header?

Reply:
good catch, fixed.